src/v/serde/serde.h

```diff
@@ -287,6 +288,9 @@ void write(iobuf& out, T t);
 template<typename T>
 void write(iobuf& out, std::vector<T> t);
 
+template<typename T, size_t Size>
```

Comment:
`std::size_t` is used in the concept, but `size_t` here

Reply:
fixed